src/components/v2v3/shared/DiffedSplits/DiffedSplitFields/DiffedSplitValue.tsx

```diff
@@ -10,17 +12,32 @@ export function DiffedSplitValue({
   splitProps: SplitProps
   diffSplit?: Split
 }) {
+  const diffSplitProps: SplitProps | undefined = diffSplit
+    ? {
+        split: diffSplit,
+        totalValue: diffSplit.totalValue,
+        projectOwnerAddress: splitProps.projectOwnerAddress,
+        currency: splitProps.currency,
+      }
+    : undefined
+
+  const splitIsZero =
+    formatSplitPercent(BigNumber.from(splitProps.split.percent)) === '0'
```

Comment:
this seems overkill, is there a different way of checking for 0? What's  split.percent type?